### ISSUE TITLE
 local costmap with two inflation layers or inflation_radius

### ISSUE THREAD

**Issue by minghaohsu410168** [ros-planning/navigation#1241]
Hi everyone,

I have tried using the move_base packages for robot navigation, and I want to set two obstacles to local costmap but different inflation radius. e.g. I have two sensors, one is LiDAR and the other is sonar. How to set the LiDAR layer costmap with inflation_radius=0.5 and sonar layer with inflation_radius=0.25?

**Comment by mikeferguson** [comment]
I don't think this is actually a supported configuration - the inflation radius is designed to be a robot-specific parameter - not really a sensor specific one. Further, by the time the inflation layer runs, it really doesn't have any information about which sensor was involved in marking a cell in the costmap.
